recipes/db12/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "DB12" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fab97a28cb7100c0c499b0e4746f7de7de1a472fc56f8376adf949efc6b055fa
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - db12 = db12.__main__:main
+
+requirements:
+  host:
+    - python >=2.7
+    - pip
+    - setuptools-scm
+  run:
+    - python >=2.7
+
+test:
+  source_files:
+    - tests
+  requires:
+    - pytest
+  imports:
+    - db12
+  commands:
+    - pytest -v
+
+about:
+  home: https://github.com/DIRACGrid/DB12
+  license: GPL-3.0-only
+  license_family: GPL
```

Looks like it's actually LGPL:
```suggestion
  license: LGPL-3.0-only
  license_family: LGPL
```